src/extension/app/plugins/publish/publish.js

```diff
@@ -50,14 +50,6 @@ export function createPublishPlugin(appStore) {
           const { host } = siteStore;
           const targetEnv = host ? 'production' : 'live';
           appStore.showToast(appStore.i18n('publish_success').replace('$1', appStore.i18n(targetEnv).toLowerCase()), 'positive', closeCallback, actionCallback, appStore.i18n('open'));
```

Comment:
Somehow I find reading _opening live_ odd... Would it feel weird to you if we left the `targetEnv` upper case?

Reply:
Happy to go with whatever you want here. I think both work

Reply:
I think we should go with the upper case names of the environments.